.mergify.yml

```diff
@@ -36,8 +36,9 @@ pull_request_rules:
     conditions:
       - label!=DNM
       - base~=^(devel)|(release-.+)$
-      - "#approved-reviews-by>=2"
       - "#changes-requested-reviews-by=0"
+      - "approved-reviews-by=@ceph/ceph-csi-contributors"
```

Comment:
I wonder if the notation should be `@ceph-csi-contributors` without the `ceph/` organization part. Did you find an example configuration file where teams are used?

Reply:
took example from https://github.com/Mergifyio/mergify-engine/issues/310#issuecomment-444419127

Reply:
I guess @ceph/ceph-csi-contributors  is working 
https://dashboard.mergify.io/github/ceph/config-editor?repositoryName=ceph-csi&pullRequestNumber=2425